src/mobile/src/api/supabase.js

```diff
@@ -0,0 +1,13 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { createClient } from '@supabase/supabase-js';
+import Constants from 'expo-constants';
+
+const ANON_KEY = Constants.manifest.extra.supabaseKey;
+const SUPABASE_URL = Constants.manifest.extra.supabaseUrl;
+
+const supabase = createClient(SUPABASE_URL, ANON_KEY, {
+  localStorage: AsyncStorage,
+  detectSessionInUrl: false,
```

Comment:
Can you explain these 2 options?

Reply:
https://medium.com/@kelvinpompey.me/things-to-look-out-for-using-supabase-with-react-native-9638b23e98c2 this might explain it